x-pack/platform/plugins/shared/observability_ai_assistant/server/service/client/index.ts

```diff
@@ -509,7 +509,9 @@ export class ObservabilityAIAssistantClient {
               this.dependencies.inferenceClient
                 .chatComplete({
                   ...options,
+                  temperature: 0.25,
```

Comment:
TODO: undo

Reply:
We should actually default to 0.25. 0 causes repetitiveness in successive responses